python/openassetio_traitgen/templates/cpp/specification.hpp.in

```diff
@@ -8,7 +8,7 @@
 
 #include <memory>
 
-#include <openassetio/TraitsData.hpp>
+#include <openassetio/trait/TraitsData.hpp>
```

Comment:
[fix] Needs a released notes change, to denote that the new runtime requirement for OpenAssetIO >= v1.0.0b1

Reply:
done in here : https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/51/commits/04dc959cd19eaff4fdf86f107cda4bd50b67da5a